db/db_creator.py

```diff
@@ -101,3 +101,14 @@ def reset_database(self) -> DBCreator:
         '''
         self.drop_records().migrate()
         return self
+
+    def get_pk_values(self, tbl_nm: str, pri_key: str) -> List[Union[int, None]]:
+        '''
+        Retrieves primary key values from the table. Only works with one primary key.
+        '''
+        pk_values = []
+        conn = self.engine.connect()
+        rs = conn.execute(f"SELECT {pri_key} FROM {tbl_nm}")
```

Comment:
Yeah, I looked at these and the problem with sqlalchemy is I didn't see any way you could add in a variable replacement that it didn't quote. 

So like if you want to add in table names or column names this is the only way I saw to do it. Values can be done using a different syntax. 

I guess they figure if you're going to be doing dynamic tables and columns you're doing it correctly.

Reply:
Yep, I've done the similar research about `drop_records`. The main precaution we can take -- without using an ORM -- is validating inputs.

Reply:
Yeah, I see what you mean. 

I'm not sure how you'd get the columns for a table. There's probably a query to figure out the primary key for a table that I could call without needing that passed in, this just seemed like a faster fix. 

As you said it's not a huge risk as both of these variables in this context are 100% safe since this these values are part of the code and not something the user can change or inject, but I see the point. 

I think I'll just leave this for now to avoid extra complexity.